examples/analog_out/cont_gen_voltage_wfm_int_clk_every_n_samples_event.py

```diff
@@ -6,35 +6,29 @@
 the specified number of samples generation is complete.
 """
 
-import numpy as np
+from analog_out_helper import create_sine_wave
 
 import nidaqmx
 from nidaqmx.constants import AcquisitionType
 
 with nidaqmx.Task() as task:
-    frequency = 10
-    amplitude = 1
-    sampling_rate = 100
-    duration = 1
-
-    # generate the time array
-    t = np.arange(0, duration, 1 / sampling_rate)
-    # Generate the sine wave
-    data = amplitude * np.sin(2 * np.pi * frequency * t)
+    sampling_rate = 1000.0
+    data = create_sine_wave(
+        frequency=10.0, amplitude=1.0, sampling_rate=sampling_rate, duration=1.0
+    )
 
     def callback(task_handle, every_n_samples_event_type, number_of_samples, callback_data):
         """Callback function for written data."""
-        print(f"transferred {number_of_samples} samples event invoked.")
+        print("Transferred N samples")
```

Comment:
This scrolls off the screen, right? How about printing the total and using `end="\r"` like you did for AI?

Reply:
True, the printing is updated as AI example.